packages/EasyErrorHandler/src/Interfaces/ErrorCodesEnumInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Interfaces;
+
+interface ErrorCodesEnumInterface
```

Comment:
We have the `AsErrorCodes` attribute now, why do we need the interface, it's unused.

Reply:
My bad. Removed